nfsv4/rpc.c

```diff
@@ -69,9 +69,13 @@ static status parse_response(rpc r, buffer b)
             break;
         }
         if (rop->parse) {
-            printf ("start decode %p %lld\n", b, buffer_length(b));            
+            if (config_boolean("NFS_DECODE_TRACE", false)) {
```

Comment:
Is there an ongoing need for this parameter? I wonder whether we could have a few debugging levels rather than many flags.